### PR TITLE
Recent updates and some cleanup

### PR DESCRIPTION
**Pull request by mschottdorf**
Deleting old files; cleanup; and making this version 0.3.0

### CODE REVIEW

**Review by zajdel — COMMENTED** [review]
Nice clean-up and encapsulation for the DLite sensor. It looks like you're labeled some lines as #pragmas, which means that tests do not exist for such lines. Is it your intention that we make tests for these lines? Test edits looked okay to me, but we may need Jonny to give them a closer look.

Documentation updates look good too.

**Review by zajdel — APPROVED** [review]

**Comment by mschottdorf** [comment]
@zajdel no; the phragma...-lines are lines added carefully whenever either criterion was fulfilled:

1.  The respective lines was tested extensively in other tests, but not countedd by travis (like the inter-process-communication in `rpc.py`).
2.  The respective line was a simple function call to a function which was already tested (e.g. the changes in `process_manager.py` ).

File: pvp/common/message.py [hunk 1]
```diff
@@ -64,12 +64,8 @@ def __init__(self, timestamp=None, loop_counter=None, breath_count=None, vals=ty
                         # otherwise just make one
                         self.timestamp = time.time()
                     else:
-                        raise e
-
-
-        # insist that we have all the rest of the vals
-        assert(all([value.name in kwargs.keys() for value in values.SENSOR.keys()]))
```
Comment (sneakers-the-rat) [inline pvp/common/message.py:71]: why was this removed?

File: pvp/common/message.py [hunk 2]
```diff
@@ -116,7 +113,8 @@ def __setitem__(self, key, value):
         elif key.lower() in self.additional_values:
             return setattr(self, key.lower(), value)
         else:
-            raise KeyError(f'No such value as {key}')
```
Comment (sneakers-the-rat) [inline pvp/common/message.py:119]: would need a bit more discussion on these changes -- these are in place to make sure we have a tight and uniform values api and i'm not sure what removing them adds

File: pvp/common/message.py [hunk 3]
```diff
@@ -144,22 +142,23 @@ def __init__(self,
...
-            try:
-                name = values.CONTROL.__members__[name]
-            except KeyError as e:
+            ls = [x for x in values.CONTROL if str(x) == name]
+            if len(ls)>0:
+                name = ls[0]
+            else:
                 logger = init_logger(__name__)
                 logger.exception(f'Couldnt create ControlSetting with name {name}, not in values.CONTROL')
-                raise e
```
Comment (sneakers-the-rat) [inline pvp/common/message.py:152]: also not sure about this change, the check seems equivalent and i don't know why pytest stuff is in the main codebase?

File: pvp/common/values.py [hunk 4]
```diff
@@ -293,10 +293,6 @@ def plot_limits(self, plot_limits):
             assert all([isinstance(x, ValueName) for x in plot_limits])
         self._plot_limits = plot_limits
 
-
-    def __setitem__(self, key, value):
```
Comment (sneakers-the-rat) [inline pvp/common/values.py:297]: why was this removed?

**Review by sneakers-the-rat — COMMENTED** [review]
my questions mostly revolve around removing the explicit uniformity checks in `message` to `values` -- why was that change made?

File: pvp/common/message.py (hunk 1 above)
Comment (mschottdorf) [reply pvp/common/message.py:71]: I felt like the asserts should be removed in the deployed code

File: pvp/common/message.py (hunk 2 above)
Comment (mschottdorf) [reply pvp/common/message.py:119]: same here. This is stuff that we tested doesn't happen in the current version of the code. We could leave this in for people who fiddle with the API - but do we want that? When in doubt, I felt like removing code that is not used

File: pvp/common/values.py (hunk 4 above)
Comment (mschottdorf) [reply pvp/common/values.py:297]: it was never used

File: pvp/common/message.py (hunk 3 above)
Comment (mschottdorf) [reply pvp/common/message.py:152]: hmm, yeah. For the first thing, `.__members__[name]` doesn't exist. The second,  we now explicitly test this exception. This is to warn pytest that an exception will come
